x-pack/plugins/maps/public/components/map/feature_tooltip.js

```diff
@@ -82,7 +89,7 @@ export class FeatureTooltip extends React.Component {
     }
 
     return (
-      <EuiFlexItem grow={false}>
+      <div className="tooltipLineDescription">
```

Comment:
You should try to follow the BEM naming scheme and use the `map` prefix for any new class names. Example for this would be:

`map[NameOfComponent]_[childComponentName]`

```suggestion
      <div className="mapFeatureTooltip__filterButton">
```

Reply:
It looks like you changed the wrong classname? And now seeing as there already is a __filterButton, this one should probably be __filterButtonWrapper

Reply:
I ended up just removing the wrapping div since that wasn't getting us anything and updating the class on the button